### ISSUE TITLE
Merge dashboards

### ISSUE THREAD

**Issue by hughjonesd** [unjournal/unjournaldata#25]
We have two shiny dashboards: uj-dashboard and DataExplorer. Other things being equal, we should only have one. Both do roughly similar things.

In future, we might want different dashboards, e.g. one for management and one for external users. But management may be better off doing stuff in Coda which doesn't require R programming expertise.

(Actually, if we could expose stuff in coda to the public, that might also be good.) 

This needs management discussion so we can work out the content and goals of the dashboard.

**Comment by hughjonesd** [comment]
See also #15.

**Comment by hughjonesd** [comment]
OK, take a look at the new "paper comparison" tab in uj-dashboard. If you think it's reasonable (with some tweaks) then we could count the old dashboard as closed and change any remaining inbound links.

**Comment by daaronr** [comment]
I agree, after a few tweaks (some important ones like 'how do I know which paper it is') I think this more or less beats the other one. 

Note there was still some 'bayesian aggregation of uncertainty' in the other one, but we will need to be more careful with that, and I was never 100% convinced it was doing it right. (But a very interesting thing to dig into).
